Project.toml

```diff
@@ -33,7 +33,7 @@ PyMNE = "6c5003b2-cbe8-491c-a0d1-70088e6a0fd6"
 UnfoldMakiePyMNEExt = "PyMNE"
 
 [compat]
-AlgebraOfGraphics = "0.6"
+AlgebraOfGraphics = "0.6, 0.7, 0.8"
```

Comment:
just to be sure - we are backwards compatible with 0.6?

Reply:
ah true, we are not

Reply:
did you remove 0.6 then? I dont see it in the new commit

Reply:
yes, i deleted it in a new commit